test/integration/test_integration_basics.py

```diff
@@ -434,13 +437,17 @@ def check_upload_expectations(fixture, parsed_json, expected_transmitted_bytes,
         expected_received_bytes)
 
   upload_bytes = 1024 * 1024 * 3
+  # TODO(#531): The dynamic-delay extension hangs unless we lower the request entity body size.
+  if "static_delay" in filter_configs:
```

Comment:
How do you feel about combining this with the above line and making it a ternary, rather than an if condition?

My 2 cents is that as is, it's possible to scan this and not notice that upload_bytes has changed from its original value.